vowpalwabbit/learner.h

```diff
@@ -32,13 +32,14 @@ enum class prediction_type_t
   multiclass,
   multilabels,
   prob,
-  multiclassprobs,
+  multiclassprobs,  // not in use (technically oaa.cc)
   decision_probs,
   action_pdf_value,
   active_multiclass
 };
 
 const char* to_string(prediction_type_t prediction_type);
```

Comment:
These functions already exist now in label_type.h and prediction_type.h

Reply:
thanks